packages/example/src/modules/program/view/ProgramList.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render } from '@testing-library/react-native';
+import { theme } from '../../../design-system/theme/theme';
+import { ThemeProvider } from '@emotion/react';
+import { SpatialNavigationRoot, DefaultFocus } from 'react-tv-space-navigation';
+import testRemoteControlManager from '../../../components/tests/helpers/testRemoteControlManager';
+import '../../../components/tests/helpers/configureTestRemoteControl';
+import { ProgramList } from './ProgramList';
+
+jest.mock('@react-navigation/native');
+
+const programSize = theme.sizes.program.portrait.width + 30;
+
+const renderWithProviders = (component: JSX.Element) => {
+  return render(
+    <ThemeProvider theme={theme}>
+      <SpatialNavigationRoot>
+        <DefaultFocus>{component}</DefaultFocus>
+      </SpatialNavigationRoot>
+    </ThemeProvider>,
+  );
+};
+
+describe('ProgramList', () => {
+  it('renders the list of item and scrolls accordingly when doing RIGHT twice', async () => {
+    const component = renderWithProviders(<ProgramList />);
+
+    const listElement = await component.findByTestId('program-list');
+    expect(listElement).toHaveStyle({
```

Comment:
- you should check that the element that you'd expect is focused
- and you should query it by its accessibility label ;)

Reply:
Done :D